firestore-bigquery-export/functions/src/events.ts

```diff
@@ -2,14 +2,38 @@ import * as eventArc from "firebase-admin/eventarc";
 
 const { getEventarc } = eventArc;
 
+/**
+ * The extension name is used to construct event types dynamically.
+ * Changing this name affects the new event type generation.
+ * @constant EXTENSION_NAME
+ */
 const EXTENSION_NAME = "firestore-bigquery-export";
 
-const getEventType = (eventName: string) =>
-  `firebase.extensions.${EXTENSION_NAME}.v1.${eventName}`;
+/**
+ * Generates both the OLD and NEW event types to maintain backward compatibility.
+ *
+ * Old Event Type: firebase.extensions.firestore-counter.v1.{eventName}
```

Comment:
I'm not sure we should be publishing both, i think we probably want a breaking change here, @pr-Mais ?

Reply:
probably yes, but should be documented well in the CHANGELOG

Reply:
I thought we had agreed in not having breaking changes. Can you explain what is the issue @cabljac @pr-Mais ?